std/process.d

```diff
@@ -2228,6 +2249,19 @@ $(D "/bin/sh").
     else version (Posix)   return environment.get("SHELL", "/bin/sh");
 }
 
+/**
+Returns the platform-specific native shell path
```

Comment:
Missing punctuation. Since it's just a constant we could also go with just `The platform-specific native shell path.`